splash/server.py

```diff
@@ -86,11 +87,12 @@ def parse_opts():
 def start_logging(opts):
     import twisted
     from twisted.python import log
-    from twisted.python.logfile import DailyLogFile
-    if opts.logfile:
-        logfile = DailyLogFile.fromFullPath(opts.logfile)
-    else:
-        logfile = sys.stderr
+    # TODO: fix this.
+    # from twisted.python.logfile import DailyLogFile
+    # if opts.logfile:
+    #     logfile = DailyLogFile.fromFullPath(opts.logfile)
+    # else:
+    logfile = sys.stderr
```

Comment:
I think we should raise an exception in Python 3 if user uses --logfile option instead of silently ignoring it. Just moving import under `if` could work: 

```
if opts.logfile:
    from twisted.python.logfile import DailyLogFile
    logfile = DailyLogFile.fromFullPath(opts.logfile)
else:
    logfile = sys.stderr
```